Tests/Recurly/Plan_Test.php

```diff
@@ -83,8 +117,36 @@ public function testUpdateXml() {
     $plan->dunning_campaign_id = '1234abcd';
 
     $this->assertEquals(
-      "<?xml version=\"1.0\"?>\n<plan><plan_code>platinum</plan_code><name>Platinum Plan</name><unit_amount_in_cents><USD>1500</USD><EUR>1200</EUR></unit_amount_in_cents><setup_fee_in_cents><USD>500</USD><EUR>500</EUR></setup_fee_in_cents><total_billing_cycles nil=\"nil\"></total_billing_cycles><tax_exempt>false</tax_exempt><tax_code>fake-tax-code</tax_code><trial_requires_billing_info>false</trial_requires_billing_info><dunning_campaign_id>1234abcd</dunning_campaign_id></plan>\n",
+      "<?xml version=\"1.0\"?>\n<plan><dunning_campaign_id>1234abcd</dunning_campaign_id><name>Platinum Plan</name><plan_code>platinum</plan_code><setup_fee_in_cents><USD>500</USD><EUR>500</EUR></setup_fee_in_cents><tax_code>fake-tax-code</tax_code><tax_exempt>false</tax_exempt><total_billing_cycles nil=\"nil\"></total_billing_cycles><trial_requires_billing_info>false</trial_requires_billing_info><unit_amount_in_cents><USD>1500</USD><EUR>1200</EUR></unit_amount_in_cents></plan>\n",
```

Comment:
i think if you use assertXmlStringEqualsXmlString you can compare if that xml is equal to plan xml and indent it to be more readable

Reply:
Yea that definitely seems like it would be better. However, it has been "decreed" that no more refactors take place (though I'm not sure how that's gonna be possible 😅) so since I was just fixing a previously broken test I didn't spend any additional time. However, my upcoming work might touch/break some tests that are similar here and I think I could make it better. We shall see!